jazz_create-serverless-service/index.js

```diff
@@ -31,15 +31,17 @@ const crud = require("./components/crud")(); //Import the utils module.
     @author:
     @version: 1.0
 **/
-
-module.exports.handler = (event, context, cb) => {
+var user_id =  undefined;
```

Comment:
it is anyways undefined, no need to specify it

Reply:
fixed